### ISSUE TITLE
UTCDateTime warnings missing cache_ok parameter

### ISSUE THREAD

**Issue by ALazyMeme** [pajbot/pajbot#1415]
```bash
/opt/pajbot/pajbot/models/stream.py:151: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
  db_session.query(Stream).filter_by(ended=False).order_by(Stream.stream_start.desc()).first()
/opt/pajbot/pajbot/models/stream.py:153: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
  self.last_stream = db_session.query(Stream).filter_by(ended=True).order_by(Stream.stream_end.desc()).first()
/opt/pajbot/pajbot/managers/deck.py:107: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
  for deck in db_session.query(Deck).order_by(Deck.last_used.desc()):
/opt/pajbot/pajbot/models/user.py:340: SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
  db_session.query(User).filter_by(login=login).order_by(User.login_last_updated.desc()).one_or_none()
```

Seems like it's safe to set to `False`, but I'm not 100% sure how to do it.

**Comment by pajlada** [comment]
Tracking https://github.com/spoqa/sqlalchemy-utc/issues/11 - not an issue we can do something about right now

**Comment by zneix** [comment]
Wasn't this fixed in #1478?

**Comment by pajlada** [comment]
> Wasn't this fixed in #1478?

Yes!

Thanks for the reminder
